public/stylesheets/main.css

```diff
@@ -380,4 +376,34 @@ h1 {
     color: #9e9e9e;
     left: auto;
     right: 0;
-}
\ No newline at end of file
+}
+
+/* Index charts */
+#charts-container {
+    padding-top: 1em;
+}
+
+#charts-nav .tabs{
+    background-color: transparent;
+    font-size: 12px;
+}
+
+#charts-nav .tabs .tab {
+    text-transform: none;
```

Comment:
`.tabs` is actually a class provided by Materialize. http://materializecss.com/tabs.html
I was overriding the style to avoid uppercase for instance